Generator/include/CGAL/point_generators_3.h

```diff
@@ -548,6 +548,27 @@ class Tetrahedron_from_c3t3_cell
 
   const Tr& tr;
 };
+
+template<typename Triangle, typename PointRange>
+class Triangle_from_soup
+{
+  typedef typename PointRange::value_type Point_3;
+  typedef typename Kernel_traits<Point_3>::Kernel          Kernel;
+  const PointRange& points;
+public:
+  Triangle_from_soup(const PointRange& pts)
+    :points(pts)
+  {}
+  typedef typename Kernel::Triangle_3 result_type;
+
+  result_type operator()(const Triangle& t) const
+  {
+    return result_type(points[t[0]],
+        points[t[1]],
+        points[t[2]]);
+  }
```

Comment:
I strongly disagree about the one liners. One shouldn't have to choose between horizontal scroll and a microscope while reading code :/

Reply:
This is 64 characters, which is well under the 80 limit (that is pretty much 100 nowadays). I don't care about one-liners, just use proper and consistent indentation and I'm happy with it.

Reply:
I guess I misunderstood, because if you put the whole function on one line it is 116 characters. If you were only talking about the definition, then that's what I did in my last commit.